tests/scripts/all-in-docker.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash -eu
```

Comment:
This is a really neat idea. Could you also provide such scripts for `tests/ssl-opt.sh` and `tests/compat.sh` ?

Reply:
Sure thing. Done for both.